libbeat/processors/add_kubernetes_metadata/_meta/fields.yml

```diff
@@ -37,6 +37,11 @@
           type: object
           description: >
             Kubernetes annotations map
+        
+        - name: replicaset
```

Comment:
If you change this to `replicaset.name` and keyword, do the tests pass?

Reply:
It does. However, since the field is not added by coredns module, we should find out whether there are more fields like this, so that the problem does not come back later when some other fields appear under replicaset.

Reply:
BTW, I will not make the changes to libbeat here, will wait until #11134 to be resolved. The changes here is just for passing the CI tests temporarily.